src/sentry/static/sentry/app/api.tsx

```diff
@@ -299,13 +299,25 @@ export class Client {
    */
   request(path: string, options: Readonly<RequestOptions> = {}): Request {
     const method = options.method || (options.data ? 'POST' : 'GET');
+
+    let fullUrl = buildRequestUrl(this.baseUrl, path, options.query);
+
     let data = options.data;
 
     if (!isUndefined(data) && method !== 'GET') {
       data = JSON.stringify(data);
     }
 
-    const fullUrl = buildRequestUrl(this.baseUrl, path, options.query);
+    // TODO(epurkhiser): Mimicking the old jQuery API, data could be a string /
+    // object for GET requets. jQuery just sticks it onto the URL as query
+    // parameters
+    if (method === 'GET' && data) {
+      const paramData = typeof data === 'string' ? data : jQuery.param(data);
```

Comment:
If you wanted to excise jQuery further, the `query-string` package could be used here.

Reply:
Yeah this will be next on my list to tackle, but the `query-string` library actually does a decent bit less than jQuery's `param`, so will need to verify we're not doing any deep object fancy-ness